scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -48,17 +47,41 @@ workflow GvsCreateFilterSet {
 
   String filter_set_info_destination_table_schema = "filter_set_name:string,type:string,location:integer,ref:string,alt:string,calibration_sensitivity:float,score:float,vqslod:float,culprit:string,training_label:string,yng_status:string"
 
-  if (!defined(git_hash) || !defined(cloud_sdk_docker) || !defined(variants_docker) || !defined(gatk_docker)) {
+  if (!defined(git_hash) || !defined(basic_docker) || !defined(cloud_sdk_docker) || !defined(variants_docker) || !defined(gatk_docker)) {
     call Utils.GetToolVersions {
       input:
         git_branch_or_tag = git_branch_or_tag,
     }
   }
 
+  String effective_git_hash = select_first([git_hash, GetToolVersions.git_hash])
+  String effective_basic_docker = select_first([basic_docker, GetToolVersions.basic_docker])
   String effective_cloud_sdk_docker = select_first([cloud_sdk_docker, GetToolVersions.cloud_sdk_docker])
   String effective_variants_docker = select_first([variants_docker, GetToolVersions.variants_docker])
   String effective_gatk_docker = select_first([gatk_docker, GetToolVersions.gatk_docker])
-  String effective_git_hash = select_first([git_hash, GetToolVersions.git_hash])
+
+  call Utils.GetReference {
+    input:
+      reference_name = reference_name,
+      basic_docker = effective_basic_docker,
+  }
+
+  String effective_interval_list = select_first([interval_list, GetReference.reference.wgs_calling_interval_list])
+  String reference_fasta = GetReference.reference.reference_fasta
+  String reference_fasta_index = GetReference.reference.reference_fasta_index
+  String reference_dict = GetReference.reference.reference_dict
+
+  # resource files - used for VETS (and VQSR)
+  String axiomPoly_resource_vcf = GetReference.reference.axiomPoly_resource_vcf
+  String axiomPoly_resource_vcf_index = GetReference.reference.axiomPoly_resource_vcf_index
+  String hapmap_resource_vcf = GetReference.reference.hapmap_resource_vcf
+  String hapmap_resource_vcf_index = GetReference.reference.hapmap_resource_vcf_index
+  String mills_resource_vcf = GetReference.reference.mills_resource_vcf
+  String mills_resource_vcf_index = GetReference.reference.mills_resource_vcf_index
+  String omni_resource_vcf = GetReference.reference.omni_resource_vcf
+  String omni_resource_vcf_index = GetReference.reference.omni_resource_vcf_index
+  String one_thousand_genomes_resource_vcf = GetReference.reference.one_thousand_genomes_resource_vcf
+  String one_thousand_genomes_resource_vcf_index = GetReference.reference.one_thousand_genomes_resource_vcf_index
```

Comment:
is it preferable to pick out every member of the struct here or perhaps just to pass the struct around and have the spots that need to access specific members drill into the struct?